### DISCUSSION TITLE
Is there a way to directly use power coming from the charger as if the battery was completely disconnected?

### DISCUSSION THREAD

**Discussion by ararezaee** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1035] [Q&A] [answered]
So what I wanna do is that charge the battery to 80% then leave the battery to discharge on its own as if the laptop was turned off and only charge back up when it reaches, lets say, 20% and in the meantime I want to just use the power coming from the charger, is that at all possible?

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @ararezaee,

This is exactly what the Charge Limiter feature in combination with the Sailing Mode feature of AlDente Pro is doing.

Best,
Matthias
